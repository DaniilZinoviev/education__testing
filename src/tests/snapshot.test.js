import { describe, test, expect } from "@jest/globals";

describe("snaphsots testing", () => {

  test("object", () => {
    const object = {
      a: 12,
      b: "testing snapshot"
    };
    expect(object).toMatchSnapshot();
  });

  test("array", () => {
    const array = [1, 2, true, { a: 15 }, "test"];
    expect(array).toMatchSnapshot();
  })
});
