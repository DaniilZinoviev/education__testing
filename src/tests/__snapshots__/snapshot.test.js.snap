// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snaphsots testing array 1`] = `
Array [
  1,
  2,
  true,
  Object {
    "a": 15,
  },
  "test",
]
`;

exports[`snaphsots testing object 1`] = `
Object {
  "a": 12,
  "b": "testing snapshot",
}
`;
